conjure-java-core/src/main/java/com/palantir/conjure/java/util/Javadoc.java

```diff
@@ -80,6 +82,10 @@ public static String render(Documentation documentation) {
                         + Javadoc.render(argument.getDocs().get()));
     }
 
+    public static String getRequestLine(HttpMethod httpMethod, HttpPath httpPath) {
+        return String.format("%s %s\n", httpMethod, httpPath);
```

Comment:
I wonder if we should take advantage of the relatively new `@apiNote` javadoc tag?
`@apiNote Endpoint {@code GET /foo/{bar}/baz}`

I don't think a trailing newline should be necessary

Reply:
Ah great I didn't know about `@apiNote`, I was wondering if there was a good tag to use, will update.

The newline is necessary (everything else gets one thanks to https://github.com/palantir/conjure-java/blob/24e6758e902f1b2c90a6e246b389ac6255be7c40/conjure-java-core/src/main/java/com/palantir/conjure/java/util/Javadoc.java#L61).

Reply:
Makes sense, thanks for pointing it out :-)